client/partition.go

```diff
@@ -89,7 +98,8 @@ func (c *PartitionClient) connect() (err error) {
 	if err != nil {
 		return fmt.Errorf("error connecting to partition leader: %w", err)
 	}
-	c.connLastUsed = time.Now().UTC()
+	c.connOpened = time.Now().UTC()
+	c.connLastUsed = c.connOpened
```

Comment:
I believe updating `connLastUsed` should be in the default case (L89). I also think it also should be reset after disconnect().

Reply:
this PR doesn't change the way `connLastUsed` is accessed / processed. they logic is the same as it was: `connLastUsed` is updated at the end or reading response: https://github.com/mkocikowski/libkafka/blob/mik/ttl/client/partition.go#L168 . there is no need to reset it after disconnect() as disconnect() sets `conn = nil` https://github.com/mkocikowski/libkafka/blob/mik/ttl/client/partition.go#L121